README.md

```diff
@@ -60,11 +60,16 @@ You can put it in the layout file where you want.
     app:layout_constraintTop_toBottomOf="@id/order_value" />
 ```
 
-Setup the type of widget view.
+Setup the configuration of widget view.
 ```kotlin
-binding.widgetView.type = ShippedSuiteType.GREEN
-binding.widgetView.isMandatory = false
-binding.widgetView.isRespectServer = true
+private val configuration = ShippedSuiteConfiguration(
+    type = ShippedSuiteType.SHIELD,
+    isInformational = false,
+    isMandatory = true,
+    isRespectServer = true
```

Comment:
just realized that we're not collecting currency and passing it to the server, are we? configuration should take `currency`. then `GET offers` should should include `currency` param (if `currency` is not included, server assumes `USD` by default).

Reply:
No, we didn't, will update it.